src/modules/vtol_att_control/standard.cpp

```diff
@@ -188,10 +188,10 @@ void Standard::update_vtol_state()
 
 					// The logic that will prevent spikes in airspeed measurements to trigger the transition
 					if ((_timestamp_transition_speed_detected == 0) &&
-					    (_airspeed_validated->calibrated_airspeed_m_s >= _params->transition_airspeed)) {
+					    (*_airspeed_smooth >= _params->transition_airspeed)) {
```

Comment:
Same here, we should consider the risks of this value being wrong

Reply:
See above